code/modules/power/cable.dm

```diff
@@ -97,9 +97,16 @@ By design, d1 is the smallest direction and d2 is the highest
 	if(invisibility != INVISIBILITY_MAXIMUM)
 		qdel(src)
 
-/obj/structure/cable/Deconstruct()
-	var/turf/T = loc
-	stored.loc = T
+/obj/structure/cable/deconstruct(disassembled = TRUE)
+	if(!(flags & NODECONSTRUCT))

```

Comment:
all these NODECONSTRUCT check in deconstruct() and obj_break() are necessary so if someone makes holo cables they don't drop real cable coils.